src/gui/plugins/component_inspector/Light.qml

```diff
@@ -289,6 +293,37 @@ Rectangle {
         id: grid
         width: parent.width
 
+        RowLayout {
+          Rectangle {
+            color: "transparent"
+            height: 40
+            Layout.preferredWidth: visualizeVisualText.width + indentation*3
+
+            Text {
+              id : visualizeVisualText
+              text: ' Visualize visual enabled'
+              leftPadding: 5
+              color: Material.theme == Material.Light ? "#444444" : "#bbbbbb"
+              font.pointSize: 12
+              anchors.centerIn: parent
+            }
+          }
+          Item {
+            Layout.fillWidth: true
+            height: 40
+
+            Loader {
```

Comment:
The switches look misaligned right now:

![image](https://user-images.githubusercontent.com/5751272/157997623-178f5e64-cce8-4a7d-9567-69d98479f82e.png)

How about aligning both to the right?

Reply:
The new UI looks good to me

![new_light_ui](https://user-images.githubusercontent.com/5751272/159065004-f60a18bb-9e9a-43b6-965d-f128a9e26402.png)